test-plugins/org.yakindu.sct.generator.cpp.test/model/test.sgen

```diff
@@ -322,34 +322,34 @@ GeneratorModel for sctunit::cpp {
 		}
 	} 
 	
-	test ConstantsVariationTests{
-		
-		feature Outlet{ 
-			targetProject = targetProject
-			targetFolder = "gtests/ConstantsVariationTest"
-			libraryTargetFolder = "libraryTarget"			
-		}		
-		
-		feature JUnitWrapper {
-			WrapToJUnit =  true
-		}			
-				
-		feature FunctionInlining {
-			inlineReactions = false
-			inlineEntryActions = false
-			inlineExitActions = false
-			inlineEnterSequences = false
-			inlineExitSequences = false
-			inlineChoices = false
-			inlineEnterRegion = false
-			inlineExitRegion = false
-			inlineEntries = false
-		}
-		
-		feature SGenModel {
-			GenerateSGen =  true
-		}
-	}
+//	test ConstantsVariationTests{
```

Comment:
Why is this test commented out? Can you fix it or remove it if it is not required anymore?